forwardproxy.go

```diff
@@ -1,25 +1,11 @@
-// Copyright 2017 Google Inc.
```

Comment:
I am concerned about this.
While I personally don't care about copyright, I don't actually own it, and I don't think we can just remove it.

Reply:
Ah... this was unintentional, as I rebuilt each file piece by piece ("as-needed"), rather than starting with the existing files. I'll restore the copyright.